packages/@guardian/source-react-components-development-kitchen/src/footer-with-contents/FooterLinks.tsx

```diff
@@ -0,0 +1,232 @@
+import { css, ThemeProvider } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import {
+	between,
+	brand,
+	brandAlt,
+	from,
+	space,
+	textSans,
+	until,
+} from '@guardian/source-foundations';
+import {
+	Column,
+	Columns,
+	Hide,
+	Link,
+	linkThemeBrand,
+} from '@guardian/source-react-components';
+import type { AnchorHTMLAttributes } from 'react';
+
+const fullWidth = css`
```

Comment:
Could we possibly extract the styles from these two components to their own files? Appreciate its not done everywhere in the dev kitchen but makes the code easier to reason with.